skills/ast-bro/SKILL.md

```
@@ -26,14 +30,16 @@ sb callers Player.TakeDamage
 
 Stop at the step that answers the question:
 
-1. **Unfamiliar directory** — `sb digest <dir>`: one-page map of every file's types and public methods.
+1. **Unfamiliar directory** — `sb digest <dir>`: one-page map of every file's types and public methods. (`digest` is an alias for `sb map --preset digest`; it accepts every `map` flag.)
    ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)